src/transactionsFactories/accountTransactionsFactory.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { assert } from "chai";
+import { Address } from "../address";
+import { TransactionsFactoryConfig } from "./transactionsFactoryConfig";
+import { AccountTransactionsFactory } from "./accountTransactionsFactory";
+
+describe("test account transactions factory", function () {
+    const config = new TransactionsFactoryConfig({ chainID: "D" });
+    const factory = new AccountTransactionsFactory({ config: config });
+
+    it("should create 'Transaction' for saving key value", async function () {
+        const sender = Address.fromBech32("erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th");
+        const keyValuePairs = new Map([[Buffer.from("6b657930", "hex"), Buffer.from("76616c756530", "hex")]]);
+
+        const transaction = factory.createTransactionForSavingKeyValue({
+            sender: sender,
+            keyValuePairs: keyValuePairs,
+        });
+
+        assert.equal(transaction.sender, "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th");
+        assert.equal(transaction.receiver, "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th");
+        assert.deepEqual(Buffer.from(transaction.data).toString(), "SaveKeyValue@6b657930@76616c756530");
```

Comment:
Can also be `assert.equal()` in this case (also see below).

Reply:
Indeed, changed.